packages/components/src/TimeFormatter/TimeFormatter.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+
+interface TimeFormatterProps {
+  /**
+   * Civil Time string to be formatted
+   * 17:45:00.000000000
+   * 11:30:00.000000000
+   */
+  readonly civilTime: string;
+
+  /**
+   * Decide to show 24 hour vs.
+   * 12 hour display
+   */
+  readonly useMilitaryTime: boolean;
+}
+
+export function TimeFormatter(props: TimeFormatterProps) {
+  const timeSplit = props.civilTime.split(":");
```

Comment:
civilTime here should be an instance of `CivilTime`. See:

https://github.com/GetJobber/atlantis/blob/master/packages/components/src/InputTime/InputTime.tsx#L2

https://github.com/tc39/proposal-temporal/blob/master/objects.md#civiltime-